### DISCUSSION TITLE
soucis installation STUdio sur Mac OS Big Sur 11.6 (Apple M1)

### DISCUSSION THREAD

**Discussion by omidmog** [marian-m12l/studio#267] [General]
Bonjour, je suis nouveau et j'essaye désespérément d'installer STUdio - Story Teller Unleashed sur Mac (Mac OS Big Sur 11.6 sur une machine Apple M1).

J'ai religieusement lue la doc de STUdio sur Git, et la doc d'hteumeuleu ( https://www.hteumeuleu.fr/mettre-ses-propres-histoires-sur-la-lunii/ )

Voici les étapes que j'ai suivi :
1 - Installation du Luniistore sur mon Mac, téléchargement de l'histoire gratuite et upload de celle-ci sur le Lunii (tout s'est bien passé)
2 - Installation de JDK (https://www.oracle.com/java/technologies/downloads/) et redémarrage
3 - Téléchargement de studio-web-ui-0.3.1 depuis Git, décompression du fichier, déplacement du dossier dans le repertoire Application
4 - Execution de studio-macos.sh avec le Terminal
5 - Execution de studio-web-ui-0.3.1.jar en clic droit j'ouvre
6 - Connexion sur le navigateur (j'ai essayé chrome ET safari )à :
- http://localhost:8080/
- http://localhost:80/
- http://127.0.0.1:8080/
- http://127.0.0.1:80/

Mais j'ai juste une erreur : "Ce site est inaccessible" sur Chrome ou "Safari ne parvient pas à se connecter au serveur" sur Safari
Quelqu'un as-t-il pu installer avec succès STUdio sur Mac ? Ou est-ce que je me suis trompé ?
Merci d'avance pour votre aide :-)

**Comment by flotoraphe** [comment]
Même soucis que toi alors que le programme se lance parfaitement sous Windows. Je me dis que cela doit être du au fait que java n'est pas encore compatible ARM mais X64. je dirais bien vivement une nouvelle version mais le projet m'a l'ait abandonné...
